packages/unlock-js/src/web3Service.ts

```diff
@@ -1055,4 +1056,39 @@ export default class Web3Service extends UnlockService {
       estimatedGasUsedUSD,
     }
   }
+
+  async getTokenIdFromTx({
```

Comment:
I wonder if this will work for any version of the lock (it should). Would you be able to add a test in the integration test suite for this? It will easily run for all versions.

Reply:
Added in integration test.